### ISSUE TITLE
metrics: iperf3: save raw results into json files

### ISSUE THREAD

**Issue by grahamwhaley** [kata-containers/tests#539]
The metrics network iperf3 tests either:
 - do not run iperf3 in `-J` json mode
 - do run with `-J`, but store the raw JSON output into a separate file from the cooked results.

Ideally I think all the iperf3 (and any other test tool we use that can generate JSON itself)  should capture the raw JSON output from the test into a section called 'raw' alongside any cooked results stored - that way we can keep all the test run result data together, and maintain as much detail as we can.

**Comment by grahamwhaley** [comment]
See #510

**Comment by GabyCT** [comment]
@grahamwhaley , I'll work in this improvement

**Comment by GabyCT** [comment]
@grahamwhaley so I did an experiment running the `iperf3 bandwidth` test enabling the `-J` and here is a finding
1. Units are reported as `bits per second` and the result will be something like this
 "bits_per_second":      4179251975.0716405
Which delivers a huge number and it is not something easy to read (are you ok with this change?). Now there is nothing we can do to modified this result as it depends from environment to environment and the results can be in `Gbps` or `Mbps` etc.
Now if I understand you correctly, you want all the `-J` results as well as the `Config`, `Result` and `env` that generates the function of `metrics_json_init` in one single `json`?

**Comment by grahamwhaley** [comment]
Hi.
 Thanks for the update. So the `-J` mode always outputs the bandwidth in bps, yes? Given that we'd like to keep the cooked `Results` section as well as a new `raw` section (where we store the uncooked output of `iperf3 -J`) in the results, then I'd suggest we pick a fixed scaling factor for translating the raw bps into the cooked Results. I would probably suggest we translate to Mbps - and (much though it irks my inner geek), we should probably ensure we [standardize](https://en.wikipedia.org/wiki/Data-rate_units) on Megabits (10^6), not mibibits (2^20).

To summarise then:
- store the raw `iperf -J` json output in a section named "raw"
- in the "Results" section, store a cooked results of raw-bps/1000000

Feel free to differ btw if you can think of something better.
OK?

**Comment by GabyCT** [comment]
@grahamwhaley yes it is always in bps, sure I will add the raw result and the Mbps result
